DEVELOPMENT.md

```diff
@@ -198,12 +198,13 @@ off-machine registry.
 Run:
 
 ```shell
-ko apply --selector knative.dev/crd-install=true -f config/
+ko apply --selector knative.dev/crd-install=true -Rf config/
```

Comment:
Aren't all CRDs in config/core?

Reply:
tl;dr I don't like this because it has monitoring in scope, which has CRDs and which should be removed (really just needs hands to drive it)

Reply:
Thank you. Yes, all CRDS are in `config/core` so updated to `ko apply --selector knative.dev/crd-install=true -Rf config/core/`.